newton/tests/test_examples.py

```diff
@@ -325,6 +325,104 @@ class TestRobotExamples(unittest.TestCase):
 )
 
 
+class TestRobotPolicyExamplesG1(unittest.TestCase):
+    pass
+
+
+add_example_test(
+    TestRobotPolicyExamplesG1,
+    name="robot.example_robot_policy",
+    devices=test_devices,
```

Comment:
@Milad-Rakhsha-NV can you only use `cuda_test_devices` for these policy examples? Running them on CPU is just super slow on the CI and probably not needed.